### PR TITLE
swi-prolog: new version 9.2.3

### PR DESCRIPTION
**Pull request by mgondan**
For the record: I need to set PYTHONHOME to run the unit tests with python embedded in prolog.

PYTHONHOME="${MINGW_PREFIX}" "${MINGW_PREFIX}"/bin/cmake.exe --build . --target test

### CODE REVIEW

File: mingw-w64-swi-prolog/PKGBUILD [hunk 1]
```diff
@@ -61,7 +52,6 @@ build() {
     -DSWIPL_PACKAGES_JAVA=OFF \
     -DSWIPL_PACKAGES_X=OFF \
     -DINSTALL_DOCUMENTATION=OFF \
-    -DVMI_FUNCTIONS=ON \
```
Comment (Biswa96) [inline mingw-w64-swi-prolog/PKGBUILD:64]: Why did you remove this?

Comment (mgondan) [reply mingw-w64-swi-prolog/PKGBUILD:64]: VMI as inline (not functions) brings a substantial speedup. I inserted the VMI switch in the last version because it somehow resolved another problem, see this discussion.

https://swi-prolog.discourse.group/t/ann-swi-prolog-9-2-0-stable-and-9-3-0-new-devel-series/7214/39

Comment (Biswa96) [reply mingw-w64-swi-prolog/PKGBUILD:64]: Could you add that in the commit message?

File: mingw-w64-swi-prolog/PKGBUILD [hunk 2]
```diff
@@ -3,15 +3,12 @@
...
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
-pkgver=9.2.2
+pkgver=9.2.3
 pkgrel=1
 pkgdesc="Prolog environment (mingw-w64)"
 arch=(any)
 mingw_arch=('mingw64' 'ucrt64' 'clang64')
 url="https://www.swi-prolog.org/"
-msys2_references=(
```
[the remark this answers is not in the document]
Comment (mgondan) [reply mingw-w64-swi-prolog/PKGBUILD:12]: Sorry, that seems to have been removed by accident. Added again.
